.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: required
 env:
   - DOCKER_COMPOSE_VERSION=1.18.0
 before_install:
+  - npm install serverless
```

Comment:
nitpick: do you mind if we use `yarn` instead of npm?

```yarn add serverless```

Reply:
I never use yard, I tried to put `yard -W add serverless` but it gives me error when trying to compile. :(

Reply:
Added `serverless` to package.json and it works now :)